stores/message_history_store.jsx

```diff
@@ -1,55 +1,43 @@
 // Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I assume this file is kept around because other components depend on it?

Reply:
Yes, exactly the CreatePost and CreateComment components. After migrate that components this file can be removed.